src/Components/Login/Login.jsx

```diff
@@ -0,0 +1,127 @@
+import React, {useState} from 'react'
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import styled from "styled-components";
+import Button from "@material-ui/core/Button";
+import FormControl from "@material-ui/core/FormControl";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import { Redirect } from 'react-router';
+import { Link } from 'react-router-dom';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import CardActions from '@material-ui/core/CardActions';
+
+const CommentForm = styled.form`
+  .textFields {
+    margin-bottom: 20px;
+    width: 100%;
+  }
+  button {
+    margin-left: auto;
+  }
+  a {
+    font-size: 0.9rem;
+  }
+`;
+
+const StyledCard = styled(Card)`
+  width: 350px;
+
+  //Center
+  margin: 0 auto;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  -webkit-box-shadow: -1px 1px 5px 1px #ccc;
+  -moz-box-shadow:    -1px 1px 5px 1px #ccc;
+  box-shadow:         -1px 1px 5px 1px #ccc;
+
+  .MuiCardHeader-title{
+    font-size: 2rem;
+  }
+  .MuiCardHeader-root{
+    padding-bottom: 5px;
+  }
+  .MuiCardHeader-subheader{
+    color: black;
+  }
+`;
+
+export default function Login({login, logout, isAuthenticated}) {
+
+  const errorEmailMessage = "Couldn't find your account";
+  const errorPasswordMessage = "Wrong password. Try again or click Forgot password";
+  
+  const [values, setValues] = useState({
+    email: '',
+    password: '',
+    errorEmail: false,
+    errorPassword: false
+  });
+
+  function validateForm() {
+    return values.email.length > 0 && values.password.length > 0;
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    login();
+  }
+
+  const handleChange = prop => event => {
+    setValues({ ...values, [prop]: event.target.value });
+  };
+
+  if(isAuthenticated){
```

Comment:
isAuthenticated will be given by a context defined higher up :). I will change this later